### ISSUE TITLE
change xdm browser monitor from displaying number

### ISSUE THREAD

**Issue by maddawg2020** [subhra74/xdm#75]
is it possible to give to option to change xdm browser monitor from displaying the number of detected video items to just showing a red dot or disable the numbers completely

![xmd](https://user-images.githubusercontent.com/62689213/77721397-c0f26c80-6fb8-11ea-8ce5-14d5ff76cd07.jpg)

**Comment by ShairiLeh** [comment]
I think i prefer number is display so we can know number of video files detected

**Comment by maddawg2020** [comment]
yeah, i'm not saying to ditch the number... just a option to display number, display a red dot or completely disable it

**Comment by subhra74** [comment]
any particular reason for not showing the numbers?

**Comment by maddawg2020** [comment]
i just find the numbers distracting... and i already know that the if i'm playing a video the link will be detected... similar to how ad blockers offer that option

**Comment by subhra74** [comment]
ok, adding to backlog

**Comment by maddawg2020** [comment]
thanks 👍
